code/modules/vehicles/wheelchair.dm

```diff
@@ -117,3 +128,9 @@
 	var/obj/vehicle/ridden/wheelchair/wheelchair_unfolded = new unfolded_type(location)
 	wheelchair_unfolded.add_fingerprint(user)
 	qdel(src)
+
+/obj/vehicle/ridden/wheelchair/proc/attach_bell(obj/structure/desk_bell/bell)
+	bell_attached = bell
+	src.generate_actions()
+	add_overlay("wheelchair_bell")
```

Comment:
You don't remove this overlay when the bell falls off.

You should instead have `update_overlays()`, which returns a list of what overlays to use, return `"wheelchair_bell"` if the bell is attached, then `update_icon()` whenever it probably changed.